### PR TITLE
[BEAM-4009] Futurize unpackaged files

### PR DESCRIPTION
**Pull request by Fematich**
This pull request prepares the unpackaged files for Python 3 support. This PR is part of a series in which all subpackages will be updated using the same approach.
This approach has been documented [here](https://s.apache.org/beam-python-3) and the first pull request in the series (Futurize coders subpackage) demonstrating this approach can be found at #5053.

R: @aaltay @tvalentyn @RobbeSneyders

### CODE REVIEW

File: sdks/python/apache_beam/pvalue.py [hunk 1]
```diff
@@ -261,7 +266,7 @@ class TaggedOutput(object):
   """
 
   def __init__(self, tag, value):
-    if not isinstance(tag, string_types):
+    if not isinstance(tag, (bytes, unicode)):
```
Comment (RobbeSneyders) [inline sdks/python/apache_beam/pvalue.py]: This is not equivalent to six.string_types.
I think this should be (str, unicode).
See https://pythonhosted.org/six/#six.string_types

**Comment by RobbeSneyders** [comment]
LGTM, Thank you!

File: sdks/python/apache_beam/pvalue.py (hunk 1 above)
Comment (Fematich) [reply sdks/python/apache_beam/pvalue.py]: Fixed!

**Comment by tvalentyn** [comment]
Could you please resolve the merge conflicts?

**Comment by aaltay** [comment]
Merged and resolved conflict at merge time. Commits:
ce07fbf17d99cd0e06625ac8b0a3bf7159800972
d2ea77fe83e4796b5fa02518c53460a861d54519

**Comment by Fematich** [comment]
run python precommit

**Pull request closed without merging**